pylint/extensions/check_elif.py

```diff
@@ -12,7 +12,7 @@ class ElseifUsedChecker(BaseTokenChecker):
     """
 
     __implements__ = (ITokenChecker, IAstroidChecker)
-    name = 'elseifused'
+    name = 'else_if_used'
```

Comment:
It is okay, let's leave it as is, this was more of a general comment than something that should be changed.